src/federated.c

```diff
@@ -162,6 +165,9 @@ void FederatedConnectionBundle_handle_tagged_msg(FederatedConnectionBundle *self
   // Take the value received over the network copy it into the payload_pool of
   // the input port and schedule an event for it.
   void *payload;
+
+  // FIXME: Which pool is this. WHere is `pool->free` called make sure that these are within critical sections
```

Comment:
typo: sed/WHere/where

Reply:
This FIXME will be solved and removed before merging